tests/BoundingBox2d.elm

```diff
@@ -68,6 +69,68 @@ intersectionConsistentWithOverlaps =
         )
 
 
+intersectionConsistentWithStrictlyOverlaps : Test
+intersectionConsistentWithStrictlyOverlaps =
+    Test.fuzz2 Fuzz.boundingBox2d
+        Fuzz.boundingBox2d
+        "'intersection' is consistent with 'strictlyOverlaps'"
+        (\first second ->
+            let
+                strictlyOverlaps =
+                    BoundingBox2d.strictlyOverlaps first second
+
+                intersection =
+                    BoundingBox2d.intersection first second
+
+                intersectionDimensions =
+                    Maybe.map BoundingBox2d.dimensions intersection
+            in
+                case ( strictlyOverlaps, intersectionDimensions ) of
```

Comment:
The indentation of the `case` block doesn’t match the rest of the file, apparently because I’m using a different version of elm-format. I’m on `elm-format-0.18 0.6.1-alpha` – should I upgrade/downgrade?

Reply:
Yeah, I'm using 0.7.0-exp. I didn't properly consider the implications on collaboration when I switched to the bleeding-edge version, for which I apologize - in the future I'll be more careful about sticking to the most common/stable version of elm-format. In the meantime, though, if you could use 0.7.0-exp for OpenSolid stuff that would be great.

Reply:
Of course man, my next commit will use the new formatting once we agree on the way forward 👌